Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj

```diff
@@ -182,6 +182,12 @@
     <Compile Include="..\Xamarin.Forms.Platform.WinRT\LayoutExtensions.cs">
       <Link>LayoutExtensions.cs</Link>
     </Compile>
+    <Compile Include="CarouselView\FlipViewControl.xaml.cs">
+      <DependentUpon>FlipViewControl.xaml</DependentUpon>
```

Comment:
The new `.xaml` files will need entries for their `.xbf` files in the UWP section of Xamarin.Forms.nuspec.